### ISSUE TITLE
Is there a way to make width of chart responsive?

### ISSUE THREAD

**Issue by mohammadfarooqi** [marmelab/EventDrops#245]
At the moment if we are on a mobile or on screen the width of the table is of set length.

Is there a way to make the width of the chart responsive? So if on mobile the view of the width gets shortened?

Does the above make sense?

Thanks

**Comment by jpetitcolas** [comment]
Yes, it makes absolutely sense! Even if it is not yet implemented. At least, it doesn't refresh width when resizing window. Yet, if you refresh your browser, it should work out of the box.

**Comment by mohammadfarooqi** [comment]
Thanks @jpetitcolas. Got it. Not an issue. Just a followup question. Is it possible to insert an image in the left side where the labels are instead of using text? 

For example where the text says 'admin-on-rest' I'd like a svg/png icon? Is that possible?

![image](https://user-images.githubusercontent.com/10905290/41103440-293dc534-6a37-11e8-8321-f6d0b915e727.png)

**Comment by jpetitcolas** [comment]
Currently not. But that's a nice addition (too much to be done! (-:). Can you create a separate issue for a better follow up?
